src/mn_setup.c

```diff
@@ -1952,8 +1953,13 @@ static setup_menu_t stat_settings3[] = {
     MI_END
 };
 
+static const char *secretmessage_strings[] = {
+    "Off", "Simple", "Count",
```

Comment:
Hm, I don't like the term "simple". Any other ideas?

Reply:
How about removing the "Simple" option? So "Count" would be just "On".

Reply:
So, we *always* count revealed secrets?

Reply:
> So, we always count revealed secrets?

Yes, but maybe some users won't like it.

Reply:
I'm sure some people wouldn't like it. How about "Off", "Report", "Count"?

Reply:
FWIW, ever since I implemented the feature in Nugget months (years?) ago, it's been _Off_, _On_ and _Count_.

Reply:
It's the same in Crispy, actually.

Reply:
So, could you change it to "Off", "On" and "Count", please?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1966/commits/ea80bf1cd7eafd5c8cd86a7d2ff1c9653148f712.